django/librarian/models.py

```diff
@@ -153,7 +157,7 @@ def reset(self, recreate=True):
         session.close()
         if recreate:
             # This will create the vector store table
-            llm.get_index(self.uuid_hex)
+            llm.get_retriever(self.uuid_hex).retrieve("?")
```

Comment:
Bug fix. llm.get_index() was not actually creating the database table. However, querying a blank index does actually create the table. This is important because otherwise, multiple Celery tasks may try to create the table simultaneously.